model/alembic/versions/37b0cb83d755_ensure_textchunks_are_in_qdrant.py

```diff
@@ -102,45 +106,54 @@ async def get_existing_chunk_ids(client: AsyncQdrantClient) -> set[str]:
 
 
 async def create_missing_points(
-    client: AsyncQdrantClient, missing_chunks: list[TextChunk]
+    client: AsyncQdrantClient, missing_chunks: list[TextChunk], session: Session
 ):
     """Create Qdrant points for missing TextChunks."""
 
     points_to_create = []
 
     for chunk in missing_chunks:
         try:
-            embedding_vector = (
-                list(chunk.embedding)
-                if hasattr(chunk.embedding, "__iter__")
-                else chunk.embedding
-            )
+            # Generate dense vector as it already overlaps from the chunking
+            dense_embeddings = config.embedding_model.embed_documents([chunk.text])
+
+            # Generate sparse vector for chunks without it
+            sparse_embedder = config.get_embedding_handler()
+            sparse_embeddings = list(sparse_embedder.embed(chunk.text))
+            sparse_embedding = sparse_embeddings[0]
 
-            point = models.PointStruct(
+            # Update chunk record
+            session.add(chunk)
+            session.commit()
+            session.refresh(chunk)
```

Comment:
I'm confused why we need qdrant + another database, but theres probably something I'm missing here